l10n_it_fatturapa_pec/models/company.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    e_invoice_user_id = fields.Many2one(
+        "res.users", "E-bill creator",
+        help="This user will be used at supplier e-bill creation.",
+        default=lambda self: self.env.user.id
```

Comment:
Rispetto alle modifiche per la 10, come mai hai tolto `groups='base.group_multi_company'`?

Reply:
@eLBati Per poter consentire di cambiare l'utente anche in caso di non multicompany

Reply:
Ok